source/js/buyers-guide/search.js

```diff
@@ -15,10 +15,54 @@ const categoryTitle = document.querySelector(`.category-title`);
 const parentTitle = document.querySelector(`.parent-title`);
 const toggle = document.querySelector(`#product-filter-pni-toggle`);
 const subcategories = document.querySelectorAll(`.subcategories`);
+const subContainer = document.querySelector(`.subcategory-header`);
 
 // TODO: turn this into a static class rather than plain JS object.
 const SearchFilter = {
   init: () => {
+    let pos = { top: 0, left: 0, x: 0, y: 0 };
+
+    const mouseDownHandler = function (event) {
+      event.preventDefault();
+      event.stopImmediatePropagation();
+      subContainer.style.cursor = "grabbing";
+      subContainer.style.userSelect = "none";
+
+      pos = {
+        left: subContainer.scrollLeft,
+        x: event.clientX,
+      };
+
+      document.addEventListener("mousemove", mouseMoveHandler);
+      document.addEventListener("mouseup", mouseUpHandler);
+    };
+
+    const mouseMoveHandler = function (event) {
```

Comment:
same naming note as before. Let's call this and all other event handlers based on what they do, not the event they're tied to.